source/neuropod/backends/torchscript/torch_tensor.hh

```diff
@@ -254,9 +259,6 @@ protected:
 };
 
 // Utility function to get an IValue from a torch tensor
-torch::jit::IValue get_ivalue_from_torch_tensor(const std::shared_ptr<NeuropodValue> &tensor)
-{
-    return std::dynamic_pointer_cast<NativeDataContainer<torch::jit::IValue>>(tensor)->get_native_data();
-}
+torch::jit::IValue get_ivalue_from_torch_tensor(const std::shared_ptr<NeuropodValue> &tensor);
```

Comment:
you could make it "inline", this is legal way to provide definition in header w/o confusing compiler

Reply:
Yup! Previously, I marked it as `inline` in an internal patch on top of Neuropod, but decided to pull it into a `.cc` file here.

I was planning on pulling more things into the `.cc` file, but taking another look, it probably makes sense to just `inline` this in the header and leave it alone